common/modal.js

```diff
@@ -5,8 +5,9 @@
 
     //TODO
     .factory('modalUtils', ["$uibModal", function ($uibModal) {
-        function showModal(params, successCB, rejectCB) {
+        function showModal(params, postRenderCB, successCB, rejectCB) {
```

Comment:
In places that `showModal` is called (that you didn't change), the `successCB` is being treated as the `postRenderCB` and the `rejectCB` is being used as the `successCB` causing errors to be thrown when a successfully action was taken.

You should add the `postRenderCB` to the end of the list of params so it can be treated as an optional parameter. This way, when it isn't defined, it will be ignored.